Improve database client options templating (#439)
netbox-community/netbox-chart: charts/netbox/README.md

| `externalDatabase.existingSecretName`           | Fetch password for external PostgreSQL from a different `Secret`    | `""`                                         |
| `externalDatabase.existingSecretKey`            | Key to fetch the password in the above `Secret`                     | `postgresql-password`                        |

> | `externalDatabase.sslMode`                      | PostgreSQL client SSL Mode setting                                  | `prefer`                                     |
> | `externalDatabase.sslRootCert`                  | PostgreSQL client SSL Root Certificate setting                      | `""`                                         |
> | `externalDatabase.connMaxAge`                   | The lifetime of a database connection, as an integer of seconds     | `300`                                        |
> | `externalDatabase.disableServerSideCursors`     | Disable the use of server-side cursors transaction pooling          | `false`                                      |
> | `externalDatabase.targetSessionAttrs`           | Determines whether the session must have certain properties         | `read-write`                                 |

Reviewer: ```suggestion
| `externalDatabase.connMaxAge`                   | The lifetime of a database connection, as an integer of seconds     | `300`                                        |
| `externalDatabase.disableServerSideCursors`     | Disable the use of server-side cursors transaction pooling          | `false`                                      |
| `externalDatabase.options`                      | Additional PostgreSQL client parameters             | `{}`                                            |
```

Reviewer 2: FYI: We are currently using these options: 

``` 
   'OPTIONS': {
      'sslmode': 'require',
      'sslcert': '/etc/ssl/certs/netbox/cloudsql/dev/cert.crt',
      'sslkey': '/etc/ssl/certs/netbox/cloudsql/dev/key.crt',
      'sslrootcert': '/etc/ssl/certs/netbox/cloudsql/dev/ca_cert.crt',
    }

```

So in the options parameter, we'd do something like?
```
    - options:
        sslmode: require
        sslcert: /etc/ssl/certs/netbox/cloudsql/dev/cert.crt
        sslkey: /etc/ssl/certs/netbox/cloudsql/dev/key.crt
        sslrootcert: /etc/ssl/certs/netbox/cloudsql/dev/ca_cert.crt
    }

```

Reviewer: @kamal-shaker This is the goal of this pull request, exactly 👍
